jupyter_ydoc/ydoc.py

```diff
@@ -113,33 +121,51 @@ def source(self, value):
             ]
         with self._ydoc.begin_transaction() as t:
             # clear document
+            # TODO: use clear
```

Comment:
Leftover?

Reply:
No. In yjs, we have a clear function. Not sure if we will have the equivalent in ypy.